.github/workflows/bot.yml

```diff
@@ -302,15 +301,9 @@ jobs:
           - flinkProfile: 'flink1.14'
             sparkProfile: 'spark3.2'
             sparkRuntime: 'spark3.2.3'
-          - flinkProfile: 'flink1.13'
-            sparkProfile: 'spark3.1'
-            sparkRuntime: 'spark3.1.3'
           - flinkProfile: 'flink1.14'
             sparkProfile: 'spark3.0'
             sparkRuntime: 'spark3.0.2'
-          - flinkProfile: 'flink1.13'
-            sparkProfile: 'spark2.4'
```

Comment:
We have flink1.14.6 and spark 2.4.8 doker image now, can just upgrade to flink1.14.

Reply:
Done